app/controllers/home_controller.rb

```diff
@@ -0,0 +1,9 @@
+class HomeController < ApplicationController
+	def index
+		@products = Product.where(retired: false)
+		@product_picker = Product.where(retired: false).where.not(id: nil).sample(3)
```

Comment:
Nice! I like the sample(3) method.
Why do you need to specify that you're picking products where id != nil? Shouldn't every product have an id?